cmd/desync/extract.go

```diff
@@ -52,6 +53,7 @@ the index from STDIN.`,
 	flags.StringSliceVar(&opt.seeds, "seed", nil, "seed indexes")
 	flags.StringSliceVar(&opt.seedDirs, "seed-dir", nil, "directory with seed index files")
 	flags.BoolVar(&opt.skipInvalidSeeds, "skip-invalid-seeds", false, "Skip seeds with invalid chunks")
+	flags.BoolVar(&opt.regenerateInvalidSeeds, "regenerate-invalid-seeds", false, "Regenerate seed indexes with invalid chunks")
```

Comment:
It might be good to add something to the command description itself. The reason is that "regenerate" could be understood in two ways.
1) Keep the data and build an index from it in memory, neither data nor index are changed on disk (worth mentioning too). This is what is implemented
2) The index is used to restore the seed data file - This is not what's happening but it'd be good to make sure it's not understood that way

Reply:
Right, I explained it in the extract description.